run-ab-platform.sh

```diff
@@ -0,0 +1,348 @@
+#!/bin/bash
+
+VERSION=0.58.0
+# Run away from anything even a little scary
+set -o nounset # -u exit if a variable is not set
+set -o errexit # -f exit for any command failure"
+
+# text color escape codes (please note \033 == \e but OSX doesn't respect the \e)
+blue_text='\033[94m'
+red_text='\033[31m'
+default_text='\033[39m'
+
+# set -x/xtrace uses a Sony PS4 for more info
+PS4="$blue_text""${0}:${LINENO}: ""$default_text"
+
```

Comment:
remove. after airbyte verion 0.64.x , they have deprecated the docker setup. we are at 1.4.1